### PR TITLE
118 show current map url

### LINKED ISSUE #118: Show the URL to the Map in the composer.
show the url to the view of the map from the composer.
After save the url shoul dbe visible and on edit as well.

### PR DESCRIPTION
**Pull request by milafrerichs**
## What does this PR do?
Shows the Map Url when the map is saved or edited. 

## Screenshot
bildschirmfoto 2017-01-24 um 11 16 21

## Related Issue
closes #118

### CODE REVIEW

File: src/css/app.css [hunk 1]
```diff
@@ -72,6 +72,25 @@ html, body {
 .layer-tree-panel {
   max-height: 94vh !important;
 }
+.map-url-wrapper {
+	position: absolute;
+  top: 0;
+  width: 50%;
+  left: 25%;
+	height: 25px;
+	background: rgb(0, 188, 212);
```
Comment (bartvde) [inline src/css/app.css]: is it not better to use a material-ui Paper as root component so we get the theming from material-ui instead?

File: src/services/geonode.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import MapConfigTransformService from 'boundless-sdk/services/MapConfigTransformService';
 import MapConfigService from 'boundless-sdk/services/MapConfigService';
 import {getCRSFToken, removeTrailingSlash} from '../helper';
-const NEW_MAP_ENDPOINT = '/maps/new/data';
+import {edit_map_endpoint, NEW_MAP_ENDPOINT} from '../constants/server'
```
Comment (bartvde) [inline src/services/geonode.js:4]: why is one uppercase, the other one lowercase?

Comment (bartvde) [reply src/services/geonode.js:4]: ah it looks like one is a function the other one is not?

**Review by bartvde — APPROVED** [review]

File: src/css/app.css (hunk 1 above)
Comment (milafrerichs) [reply src/css/app.css]: sure. did not knew what paper is :) but sounds good.

File: src/services/geonode.js (hunk 2 above)
Comment (milafrerichs) [reply src/services/geonode.js:4]: yes, not sure if constant is the right place for it. probably not. do you have a good suggestion?

**Comment by milafrerichs** [comment]
bildschirmfoto 2017-01-24 um 17 10 36

File: src/components/mapUrl.jsx [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
+    const style = {
+      background: background,
+      color: textColor,
+      padding: '5px'
+    }
     return (
-      <div className={'map-url-wrapper ' + showHideClass}>
-        <a href={this.props.url} className='map-url'>{this.props.url}</a>
-      </div>
+      <Paper className={'map-url-wrapper ' + showHideClass} children={content} style={style}/>
```
Comment (bartvde) [inline src/components/mapUrl.jsx:15]: you might need to add zIndex={0} if you don't want a "border"

File: src/components/mapUrl.jsx [hunk 4]
```diff
@@ -1,16 +1,29 @@
 import React from 'react';
+import Paper from 'material-ui/Paper'
+import muiThemeable from 'material-ui/styles/muiThemeable';
 
-export default class MapUrl extends React.Component {
+export class MapUrl extends React.Component {
   render() {
+    const background = this.props.muiTheme ? this.props.muiTheme.palette.primary1Color : '#fff';
```
Comment (bartvde) [inline src/components/mapUrl.jsx]: this should not be needed?

Comment (bartvde) [reply src/components/mapUrl.jsx]: basically I would not expect the need to pass on any style config at all here

Comment (milafrerichs) [reply src/components/mapUrl.jsx]: it does make test easier and if you want to use the component without the material-ui then maybe :)

Comment (bartvde) [reply src/components/mapUrl.jsx]: how would you do that if it depends on Paper anyway?

Comment (milafrerichs) [reply src/components/mapUrl.jsx]: right. that's true :)

**Pull request merged**